src/templates/playgroundCreateCollectionTemplate.ts

```diff
@@ -2,14 +2,11 @@ const template = `/* global use, db */
 // MongoDB Playground
 // Use Ctrl+Space inside a snippet or a string literal to trigger completions.
 
-const database = 'NEW_DATABASE_NAME';
-const collection = 'NEW_COLLECTION_NAME';
-
 // Create a new database.
-use(database);
+use('CURRENT_DATABASE');
```

Comment:
Did Gaurab request it?

Reply:
> Additionally, we should also add an icon for Playgrounds here for so users can directly jump to Playgrounds from the connection tree. If users are navigating at the DB level, we can pre-populate playgrounds with `use(<DB_Name>);` and if they are clicking from the collection level, we can additionally include a simple db.<collection_name>.findOne().

This was the request from Guarab; I thought I'd just reuse this template but just noticed the `findOne()` detail so will likely create a new one. Other than that, just thought it seemed nice to make the template shorter by moving the variables in but I could just leave it as is.

Reply:
Setting names as variables first was intentional to draw users' attention that those are placeholders and need to be changes according to their namespase.